### DISCUSSION TITLE
Error from the Data Importer. CSV cannot be converted

### DISCUSSION THREAD

**Discussion by TailscaleNAS** [firefly-iii/firefly-iii#8747] [Firefly III]
Hi everyone, 

I try to import a CSV downloded from my bank but I get this error during the conversion:

![image](https://github.com/firefly-iii/firefly-iii/assets/146654337/4366e442-e709-496c-966c-d6ae989f9123)

I removed any extra lines and I kept my CSV as simple as I can, but I cannot get rid of the error.
Last version of both Data Interpreter and Firefly III installed.

**Comment by JC5** [comment]
Mmm, curious. I'd like to see more details on the errors.

https://docs.firefly-iii.org/how-to/general/debug/

**Reply by TailscaleNAS** [reply]
[
[firefly_iii_importer.txt](https://github.com/firefly-iii/firefly-iii/files/14854022/firefly_iii_importer.txt)
](url)


Here a log file with all the details, hope you can help me @JC5

**Reply by JC5** [reply]
I see that it crashes, but I can't say why. Try select the option to convert the file to utf8, it's in the Configuration screen.

**Reply by TailscaleNAS** [reply]
Already tried. Same error. Is it possible to see somewhere in the documentation how an excel file should look before the conversion? I can try to remove as much as possible (even though I removed already a lot, I have like 4 columns, amount debit / credit, description and another one )

**Reply by TailscaleNAS** [reply]
Actually when I activate that function, I don't get any error but it shows like in the screenshot that I posted without any type of message that the conversion is over. Just the same buttons but no message. It is fully bugged.

**Reply by JC5** [reply]
Sorry, perhaps try to convert it to utf8 using something like Notepad++?

**Reply by TailscaleNAS** [reply]
Update: i made it work by removing the description column, I had to manually write the description for all the transaction. I believe the problem is that the description cell is too long. I will try to make it shorter and see if something changes. 

It would be nice to have a rule where for a specific description the transaction get directly assigned to a specific category
